dolphinscheduler-ui/src/views/security/alarm-instance-manage/use-form.ts

```diff
@@ -72,11 +72,15 @@ export function useForm() {
   const getUiPluginsByType = async () => {
     if (state.pluginsLoading) return
     state.pluginsLoading = true
-    const plugins = await queryUiPluginsByType({ pluginType: 'ALERT' })
-    state.uiPlugins = plugins.map((plugin: IPlugin) => ({
-      label: plugin.pluginName,
-      value: plugin.id
-    }))
+    try {
+      const plugins = await queryUiPluginsByType({ pluginType: 'ALERT' })
+      state.uiPlugins = plugins.map((plugin: IPlugin) => ({
+        label: plugin.pluginName,
+        value: plugin.id
+      }))
+    } catch (e) {
+      state.pluginsLoading = false
+    }
     state.pluginsLoading = false
```

Comment:
Setting pluginsLoading to false should be within the try block.

Reply:
ok，thanks